Heston/HestonCall.cs

```diff
@@ -264,6 +264,44 @@ public static double HestonCallPrice(double kappa, double theta, double rho, dou
             return call;
         }
 
+        public static double HestonPercentageCallPrice(double kappa, double theta, double rho, double v0, double sigma, double s0, double T, double K, double r, double q, double? timeToPaymentDate = null)
```

Comment:
rinomina il metodo in heston call undiscounted e usalo nella funzione HestonCallPrice (dove chiamerai HestonCallUndiscounted e applicherai il discounted come veniva fatto prima)